Algorithm.Python/MeanVarianceOptimizationAlgorithm.py

```diff
@@ -68,7 +68,7 @@ def coarseSelector(self, coarse):
 
     def OnOrderEvent(self, orderEvent):
         if orderEvent.Status == OrderStatus.Filled:
-            self.Debug(orderEvent.ToString())
+            self.Debug(orderEvent)
```

Comment:
was this intentional/required?

Reply:
@AlexCatarino suggested that `self.Debug(orderEvent)` is the way, see above.